terraform/aptos-node/aws/kubernetes.tf

```diff
@@ -155,6 +155,68 @@ resource "helm_release" "validator" {
   }
 }
 
+resource "helm_release" "logger" {
+  count       = var.enable_logger ? 1 : 0
+  name        = "${local.workspace_name}-log"
+  chart       = "${path.module}/../../helm/logger"
+  max_history = 10
+  wait        = false
+
+  values = [
+    jsonencode(var.logger_helm_values),
+    jsonencode({
+      logger = {
+        name = "aptos-logger"
+      }
+      chain = {
+        name = var.chain_name
+      }
+      serviceAccount = {
+        create = false
+        name = "${local.workspace_name}-aptos-node-validator"
+      }
+    }),
+  ]
+
+  set {
+    name  = "timestamp"
+    value = timestamp()
+  }
+}
+
+resource "helm_release" "monitoring" {
+  count       = var.enable_monitoring ? 1 : 0
+  name        = "${local.workspace_name}-mon"
+  chart       = "${path.module}/../../helm/monitoring"
+  max_history = 10
+  wait        = false
+
+  values = [
+    jsonencode(var.monitoring_helm_values),
+    jsonencode({
+      chain = {
+        name = var.chain_name
+      }
+      validator = {
+        name = var.validator_name
+        releaseName = local.workspace_name
```

Comment:
what is releaseName? We should try not to introduce too many mandatory configurations if it's confusing to operators

Reply:
nice catch, it was testing stuff, we don't need it